holdings_maintenance/pds4/pds4linkshelf.py

```
@@ -0,0 +1,1219 @@
+                        help='The path to the root directory of a bundle.')
+
+    parser.add_argument('--log', '-l', type=str, default='',
+                        help='Optional root directory for a duplicate of the ' +
+                             'log files. If not specified, the value of '      +
+                             'environment variable "%s" ' % LOGROOT_ENV        +
+                             'is used. In addition, individual logs are '      +
+                             'written into the "logs" directory parallel to '  +
+                             '"holdings". Logs are created inside the '        +
+                             '"pdslinkshelf" subdirectory of each log root '   +
+                             'directory.'
+                             )
```

Flake8 (7.2.0)

[error] closing bracket does not match visual indentation

(E124)